### DISCUSSION TITLE
Twtr.Plus instance?

### DISCUSSION THREAD

**Discussion by WorksAtCisco** [mastodon/mastodon#25408] [Q&A]
Hey everyone! I run a corporate account on Mastodon.Social, but one of my teammates found a possibly fake account on a server called twtr.plus. I can't access that server nor does the account or server in question show up in searches. Is anyone else familiar with this instance or server? The account in question is @talossecurity@twtr.plus and it appears to be a bot but I don't want anything impersonating us. Thanks for your help in advance.

**Comment by trwnh** [comment]
twtr.plus looks to be a defunct / offline instance of BirdSiteLive, which is itself defunct software which mirrored Twitter accounts on-demand. It's not impersonation, it's literally just a 1:1 one-way bridge between fedi and twitter, in the same vein as services which create RSS feeds from twitter accounts. You would put in a twitter username and it would generate a proxy actor for you to follow. There's generally nothing that can really be done about this; what you are seeing is cached data from when twtr.plus was still alive. The most you could do is maybe clear that cache, or ask the admins of mastodon.social to clear that cached account.
